metadata-io/src/main/java/com/linkedin/metadata/entity/cassandra/CassandraAspectDao.java

```diff
@@ -526,17 +495,75 @@ public long saveLatestAspect(
     }
     // Save oldValue as the largest version + 1
     long largestVersion = ASPECT_LATEST_VERSION;
+    BatchStatement batch = BatchStatement.newInstance(BatchType.LOGGED);
```

Comment:
Does this statement need to be LOGGED? I thought `urn`+`aspectName` would always be on a single partition, thus avoiding the need to log (and its performance penalty).

I'm taking this as my inspiration as I have very little Cassandra knowledge: https://www.baeldung.com/java-cql-cassandra-batch#2-single-partition-batch